stream-chat-android-state/src/main/java/io/getstream/chat/android/offline/plugin/logic/channel/thread/internal/ThreadStateLogic.kt

```diff
@@ -16,47 +16,73 @@
 
 package io.getstream.chat.android.offline.plugin.logic.channel.thread.internal
 
+import io.getstream.chat.android.client.extensions.internal.NEVER
 import io.getstream.chat.android.client.models.Message
+import io.getstream.chat.android.client.utils.SyncStatus
 import io.getstream.chat.android.offline.plugin.state.channel.thread.internal.ThreadMutableState
 
 /**
  * The logic of the state of a thread. This class contains the logic of how to
  * update the state of the thread in the SDK.
+ *
+ * @property mutableState [ThreadMutableState]
  */
-internal interface ThreadStateLogic {
```

Comment:
As far as this Interface is Internal and only has one implementation, it is unnecessary.
When we will plan to use it from a different module and we really need the interface from there, we can pull method members up to an interface and put the interface on the proper module it needs to live

Reply:
The internal interface was only to avoid keeping it public before the time... but it won't be hard to create a different interface when it is needed, so no need to stuck this PR over this, let's move on.